third-party/rspamd/src/main/java/org/apache/james/rspamd/model/AnalysisResult.java

```diff
@@ -94,15 +102,18 @@ public String getDescription() {
     private final float score;
     private final float requiredScore;
     private final Optional<String> desiredRewriteSubject;
+    private final boolean hasVirus;
 
     public AnalysisResult(@JsonProperty("action") Action action,
                           @JsonProperty("score") float score,
                           @JsonProperty("required_score") float requiredScore,
-                          @JsonProperty("subject") Optional<String> desiredRewriteSubject) {
+                          @JsonProperty("subject") Optional<String> desiredRewriteSubject,
```

Comment:
Do we still need `@JsonProperty` here?
I see `AnalysisResultDeserializer` already